azuredriver/service/azureresourcemanager.py

```diff
@@ -1,41 +1,47 @@
+'''Azure resource manager module'''
 import logging
 from pathlib import Path
 import re
 import os
 import uuid
 import json
 import fnmatch
-from azuredriver.service.common import DELETE_REQUEST_PREFIX
-from azuredriver.service.common import CREATE_REQUEST_PREFIX, build_request_id
 from ignition.model.lifecycle import LifecycleExecuteResponse
 from ignition.service.templating import ResourceTemplateContextService, Jinja2TemplatingService
-from ignition.service.resourcedriver import InfrastructureNotFoundError, ResourceDriverError, InvalidRequestError
-from ignition.service.framework import interface
-from azuredriver.service.topology import AZUREAssociatedTopology
+from ignition.service.resourcedriver import InfrastructureNotFoundError, ResourceDriverError
+from ignition.service.resourcedriver import InvalidRequestError
 from azure.core.exceptions import *
 from azuredriver.model.exceptions import *
```

Comment:
import StackNotFoundError instead of wildcard

Reply:
Done